test/remote_coffee_slack/members_test.rb

```diff
@@ -1,7 +1,8 @@
 require "test_helper"
 
-class RemoteCoffeeSlack::MembersTest  < Minitest::Test
+class RemoteCoffeeSlack::MembersTest < Minitest::Test
   def setup
+    RemoteCoffeeSlack.config.members_per_group = 2
```

Comment:
Why did you add this line? The default value must be 2, shouldn't be necessary to set a default parameter.

Reply:
This test is a flaky one.
Sometimes this value is 4 and it fails